jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/FifoPanel.kt

```diff
@@ -34,18 +30,23 @@ class FifoPanel {
             setDescription(message("sqs.message.group_id.tooltip"))
             installOn(groupContextHelp)
         }
-    }
-
-    private fun setFields() {
         deduplicationId.emptyText.text = message("sqs.required.empty.text")
         groupId.emptyText.text = message("sqs.required.empty.text")
+        clear()
     }
 
     fun validateFields(): List<ValidationInfo> = listOfNotNull(
         validateDedupeId(),
         validateGroupId()
     )
 
+    fun clear(isSend: Boolean = false) {
+        deduplicationId.text = UUID.randomUUID().toString()
+        if (!isSend) {
+            groupId.text = ""
```

Comment:
Do we do a default value for groupId?

Reply:
No, and I think it wouldn't make sense to since that should be something defined in the application. The change here is to not clear it on send so it's easy to send more messages.